R/NPSForVeg_Class_def.R

```diff
@@ -40,6 +42,7 @@ setClass(Class="NPSForVeg",           ### Name of the Class
           ShSeedPlotSize="numeric",   ### Number and area of the plots where shub seedings are sampled in m^2
           VPlotSize="numeric",        ### Number and area of the plots where vines are sampled in m^2
           HPlotSize="numeric",        ### Number and area of the plots where herbs are sampled in m^2
+          CWDPlotSize="numeric",      ### Number, slope, and length of transects where CWD is sampled
```

Comment:
Should `CWDPlotSize` now be `CWDTransectSize` and be a `data.frame` rather than `numeric`?

Reply:
Oops. Yep, forgot to update that section after I adjusted the approach for organizing transect information. Revised pull request coming soon!